src/index.ts

```diff
@@ -5,7 +5,12 @@ import { Company } from './company';
 import { Database } from './database';
 import { Datatype } from './datatype';
 import { _Date } from './date';
-import type { Definitions } from './definitions';
+import type {
+  Definitions,
+  DefinitionTypes,
+  LocaleDefinition,
+} from './definitions';
+import { definitions } from './definitions';
```

Comment:
Maybe we should use CONSTANT_CASE so we do not confuse with the Faker definitions.

```suggestion
import { definitions as DEFINITIONS } from './definitions';
```

Reply:
Only here or also the exported values?

Reply:
I think it makes sense also for the export